openedx/openedx-platform
# Accel hawthorn stage

**Description:** Register bookmarks celery tasks for fix bug.

**Youtrack:** [ticket](https://youtrack.raccoongang.com/issue/ACCEL-131)

**Testing instructions:**

1. Open CMS and SignIN
2. Create course and add Section/Subsection/Unit
3. Open LMS and login
4. Enroll on created course and add this unit in bookmarks on the "course" page LMS
5. Change names for Section/Subsection/Unit on Section1/Subsection1/Unit1
6. Check "Bookmarks" page
